index.php

```diff
@@ -0,0 +1,108 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>User Rank</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">

```

Comment:
Again, this is an external asset, so every time someone visits your tool, their IP address gets announced to bootstrapcdn.com. As the developer community of a large global website which is often at odds with the local government or other forces, we prefer to be cautious about making our users trackable.

Reply:
@tgr, I am sorry about the pull request and external use of asset next time I'll notice, Thanks.